bigquery/tables.rb

```diff
@@ -144,10 +138,9 @@ def import_table_data_from_cloud_storage project_id:, dataset_id:, table_id:,
   # table_id     = "ID of the table to import data into"
   # storage_path = "Storage path to file to import, eg. gs://bucket/file.csv"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.